distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table_Conversions.enso

```diff
@@ -6,36 +6,49 @@ import Standard.Base.Errors.Unimplemented.Unimplemented
 
 import project.Data.Match_Columns.Match_Columns
 import project.Data.Table.Table
-import project.Delimited.Delimited_Format.Delimited_Format
 import project.Errors.Invalid_JSON_Format
-import project.Internal.Delimited_Reader
-import project.Internal.Delimited_Writer
 import project.Internal.Parse_To_Table
+import project.Internal.Widget_Helpers
 
-## Converts a Text value into a Table.
-
-   The format of the text is determined by the `format` argument.
-
-   Arguments:
-   - that: The text to convert.
-   - format: The format of the text.
-   - on_problems: What to do if there are problems reading the text.
-Table.from (that : Text) (format:Delimited_Format = Delimited_Format.Delimited '\t') (on_problems:Problem_Behavior=Report_Warning) =
-    case format of
-        _ : Delimited_Format -> Delimited_Reader.read_text that format on_problems
-        _ -> Unimplemented.throw "Table.from is currently only implemented for Delimited_Format."
+## GROUP Standard.Base.Conversions
+   Expand a column of objects to a new set of columns.
 
-## Converts a Table into a Text value.
+    Arguments:
+    - column: The column to expand.
+    - fields: The set fields to expand. If `Nothing` then all fields are added.
+    - prefix: Prefix to add to the column names. If `Nothing` then the column
+      name is used.
+@column Widget_Helpers.make_column_name_selector
+Table.expand_column : Text | Integer -> Vector | Nothing -> Text | Table -> Table ! Type_Error
```

Comment:
```suggestion
Table.expand_column : Text | Integer -> Vector | Nothing -> Text | Table -> Table ! No_Such_Column | Index_Out_Of_Bounds
```

I don't think we should list `Type_Error` there - we usually don't in other places - technically it can happen in almost every function (not all have typechecks yet but more and more do - I don't think annotating it everywhere will help).

On the other hand, the `column` argument may point to a column that does not exist - it may be worth to note these errors are possible.

Reply:
Included Type_Error as it is currently thrown by `from_objects` is the value is of an unsupported tyep.